interface/HHAnalyzer.h

```diff
@@ -105,20 +109,21 @@ class HHAnalyzer: public Framework::Analyzer {
         void fillTriggerEfficiencies(const Lepton & lep1, const Lepton & lep2, Dilepton & dilep);
 
         // global event stuff (selected objects multiplicity)
-        BRANCH(nJets, unsigned int);
+        ONLY_NOMINAL_BRANCH(nJets, unsigned int);
         BRANCH(nJetsL, unsigned int);
```

Comment:
Sorry, indeed for the leptons it matters, but for the jets because of https://github.com/cp3-llbb/HHAnalysis/pull/125/files#diff-6cdb42586ea6220d1c53fa646556a54cR615 there is no difference between nJets and nJetsL?